src/components/UI/Text/InputNumber.vue

```diff
@@ -257,7 +262,14 @@ export default {
     },
     keyPress(event) {
       // invoked when a key is pressed
-      if (this.maxLength != null && this.localValue.length == this.maxLength) {
+      // character key not accepted
+      event = event || window.event;
+      var charCode = (typeof event.which == "undefined") ? event.keyCode : event.which;
```

Comment:
You're using `event.which` here.
If you read the docs for this, they give a warning that this is a non standard and should not be used in production as it's result is highly variable. See this - https://developer.mozilla.org/en-US/docs/Web/API/UIEvent/which

Did you find this piece of code online somewhere? It's okay if you did, but make you understand what's happening.

Reply:
Yes, I did, I went through the code I did not knew about the warning.

Reply:
Removed it as we only need the keycode to compare the keypress.
Thank you for your feedback, will surely keep these things in mind in the future.